.github/workflows/release.yml

```diff
@@ -46,8 +58,8 @@ jobs:
           body_path: ./tmp/checksums.txt
           files: |
             ./tmp/release.yml
-            ./tmp/package.yml
-            ./tmp/package-metadata.yml
+            carvel-artifacts/packages/secretgen-controller.carvel.dev/package.yml
+            carvel-artifacts/packages/secretgen-controller.carvel.dev/metadata.yml
```

Comment:
In today `secretgen-controller` releases, file is called package-metadata.yml

Reply:
Also, if we see the checksum in already releases, it is something like this:
```
5c95408656d898f724df7a29ec4052762a94711a8677637625fbb770af2f3cf4  release.yml
aa1326743e61eb8e9e1122668a9434961a47c36982b65e4d33cc08eab0618dd5  package.yml
55338cd490a6932c857e2035ed552b0a21e07bd2f0a46d4f5d0d2419b2323333  package-metadata.yml
```

Reply:
I wouldn't consider the naming a blocker. Given that this is a release artifact I just wanted it be very explicit that it was a package metadata file and not some other kind of release metadata. For the purposes of functionality, it shouldn't matter too much. Anyone vendiring the file's will probably have to update, but that's ok. I don't think there are many people doing that.

Reply:
Changed the file name to `package-metadata.yml` as it was earlier.